config/zephyr/Kconfig

```diff
@@ -175,6 +175,21 @@ config CHIP_OPERATIONAL_TIME_SAVE_INTERVAL
 	  precisely operation time in case of device reboot and maximizing flash memory
 	  lifetime.
 
+config CHIP_MALLOC_SYS_HEAP
```

Comment:
Are we going to enable using it somewhere? I see that it is disabled by default and we don't enable it anywhere.

Reply:
I think so, but it's kind of experimental for now, so we should probably run some tests before we do so.